src/test/java/jUnit/JsonParserTest.java

```diff
@@ -12,27 +11,35 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static jUnit.Utils.Constants.FILE_NAME_LIST;
 import static org.junit.jupiter.api.Assertions.*;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JsonParserTest {
 
-    static Cart initialCart;
-    static JsonParser parser;
+    Cart initialCart;
+    JsonParser parser;
+
+    List<String> fileNameList = Stream.of(
+            "vadim-cart",
+            "vadim1-cart",
+            "vadim2-cart")
+            .collect(Collectors.toList());
 
     @BeforeAll
-    static void setUp() {
+    void setUp() {
         parser = new JsonParser();
```

Comment:
as agreed, we'll instantiate Cart object in BeforeAll method for convenience purpose
as for JsonParser - let's declare and instantiate it on a class level